integration/docker/entrypoint.sh

```diff
@@ -12,12 +12,16 @@
 
 set -e
 
-if [[ $# -ne 1 ]]; then
-  echo 'expected one argument: "master", "worker", or "proxy"'
+EXPECTED='"master [--no-format]", "worker [--no-format]", or "proxy"'
```

Comment:
it would be good to define a `USAGE` string to print whenever there's a parsing error, similar to what we do for other scripts in `bin/`